README.md

```diff
@@ -154,15 +156,15 @@ Write Ruby code that will create an instance of a person.
 Your answer...
 
 ```ruby
-Replace this with your answer
+spiderman = Person.new("Peter Parker", "15")
 ```
 
 Write Ruby code that will query for any person that is 15 years of age.
 
 Your answer...
 
 ```ruby
-Replace this with your answer
+fifteen = Person.find_by(age < 15)
 ```
```

Comment:
the question prompt may be a little ambiguous but i think you want `where` but i get using find by since it says any person instead of every person. the argument to either in any case should be `age: 15`